js/components/portal-dashboard/account-owner.tsx

```diff
@@ -12,7 +12,7 @@ export class AccountOwnerDiv extends React.PureComponent <IProps> {
       <div className={css.accountOwner} data-cy="account-owner">
         <div className={css.accountOwnerName}>{this.props.userName}</div>
         <div className={css.accountOwnerIcon}>
```

Comment:
Remove class name for div since there is no accountOwnerIcon in css

Reply:
Good catch.  That whole div can come out, in fact.  Fixed.